src/application.coffee

```diff
@@ -635,7 +635,7 @@ getRemoteState = (uuid, apiKey) ->
 # TODO: Actually store and use app.environment and app.config separately
 parseEnvAndFormatRemoteApps = (remoteApps, uuid, apiKey) ->
 	appsWithEnv = _.mapValues remoteApps, (app, appId) ->
-		utils.extendEnvVars(app.environment, uuid, appId, app.name, app.commit)
+		utils.extendEnvVars(app.environment, uuid, apiKey, appId, app.name, app.commit)
```

Comment:
You use a passed in `apiKey` here but then do a `utils.getConfig('deviceApiKey')` below, is that correct?

Reply:
Yes, I guess I'll call it deviceApiKey here for consistency.